include/tunnler/room.h

```diff
@@ -77,6 +78,15 @@ class Room final {
     /**
      * Sends the information about the room, along with the list of members
      * to every connected client in the room.
+     * The packet has the structure:
+     * <MessageID>ID_ROOM_INFORMATION
+     * <RakString> room_name
+     * <size_t> member_slots: The max number of clients allowed in this room
+     * <size_t> num_members: the number of currently joined clients
```

Comment:
Any reason for using `size_t` ? (size_t usually reflects the size of a pointer, we'll likely only ever need uint8_t or uint16_t, uint32_t tops..)